index.html

```diff
@@ -172,24 +175,37 @@ <h3 class="card-heading">Skills</h3>
 
   <!-- contact section -->
   <section class="contact">
```

Comment:
- [ ] your contact section and footer design is very different from the figma design blelow i have drawn lines which you can use to see the placement of each element which should help you pattern your design structure before resubmission :+1: 
- [ ] your form is also looking very different from the figma template please look into ficing the design accoringly :+1:
- [ ] It is important to make sure your design closely matches the given design template as this shows how much attention you put into achieving a given design please take note of these requirements and adopt them to your design before requesting a new review :+1: :grin:  
- figma design
![fig-ff](https://user-images.githubusercontent.com/89055502/236406870-c3e9f983-5c03-4a9d-ab50-cb0250700a93.png)
- your design
![fig-ss](https://user-images.githubusercontent.com/89055502/236406968-5784ec74-5ee6-4d60-896b-e072a94ff1e8.png)